roles/openshift_docker_gc/tasks/main.yaml

```diff
@@ -4,7 +4,7 @@
     name: openshift_master
     tasks_from: ensure_nodes_matching_selector.yml
   vars:
-    openshift_master_ensure_nodes_selector: "{{ r_docker_gc_node_selectors }}"
+    openshift_master_ensure_nodes_selector: "{{ r_docker_gc_node_selectors | map_to_pairs }}"
```

Comment:
Is this supposed to be in this patch set?

Reply:
Yes.